lib/node_modules/@stdlib/string/prev-grapheme-cluster-break/lib/main.js

```diff
@@ -71,6 +71,12 @@ function prevGraphemeClusterBreak( str, fromIndex ) {
 	var cp;
 	var i;
 
+	var regexEmoji = new RegExp("\\p{Extended_Pictographic}", "ug");
+	var regexEmojiMatching = str.match(regexEmoji)
+	if (regexEmoji.test(str) && regexEmojiMatching && regexEmojiMatching.length === 1) {
```

Comment:
You're not conforming to our style conventions which can be readily observed in this source file.